Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/RefreshViewTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.RefreshView)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.None, 0, "Refresh View Tests", PlatformAffected.All)]
+	public class RefreshViewTests : TestContentPage
+	{
+		RefreshView _refreshView;
+		public RefreshViewTests()
+		{
+		}
+
+		protected override void Init()
+		{
+			Title = "Refresh View Tests";
+
+			_refreshView = new RefreshView()
+			{
+				Content = new ScrollView(),
+				Command = new Command(async () =>
+				{
+					await Task.Delay(2000);
+					_refreshView.IsRefreshing = false;
```

Comment:
can we also make sure this works without being after a task delay?

Maybe we could even add a check in here when the command is first executed that will throw an exception if *_refreshView.IsRefreshing* is true?